### PR TITLE
add filters argument

### LINKED ISSUE #37: harp reader is missing mandatory parameter filters
class init is missing the mandatory filters parameter:
https://github.com/metno/pyaro-readers/blob/1e9dc627fbb4202a45b52e36510bc32bb03f0a52/src/pyaro_readers/harpreader/harpreader.py#L31

should look like https://github.com/metno/pyaro-readers/blob/1e9dc627fbb4202a45b52e36510bc32bb03f0a52/src/pyaro_readers/aeronetsunreader/AeronetSunTimeseriesReader.py#L60-L67

Only the filters parameter is mandatory.

### PR DESCRIPTION
**Pull request by jgriesfeller**

### CODE REVIEW

File: src/pyaro_readers/harpreader/harpreader.py [hunk 1]
```diff
@@ -109,18 +141,29 @@ def _get_data_from_single_file(
         """
         dt = xr.open_dataset(file)
 
+        if dt.attrs.get("Conventions", None) != "HARP-1.0":
+            raise ValueError(f"File {file} is not a HARP file.")
+            return False
```
Comment (thorbjoernl) [inline src/pyaro_readers/harpreader/harpreader.py]: I think this return statement is unreachable and therefore superfluous.

File: src/pyaro_readers/harpreader/harpreader.py [hunk 2]
```diff
@@ -109,18 +141,29 @@ def _get_data_from_single_file(
         """
         dt = xr.open_dataset(file)
```
Comment (thorbjoernl) [inline src/pyaro_readers/harpreader/harpreader.py]: Should use context manager.

File: src/pyaro_readers/units_helpers.py [hunk 3]
```diff
@@ -0,0 +1,104 @@
+import pandas as pd
+
+
+class UnitConversionError(ValueError):
+    pass
+
+
+# from pyaerocom.tstype import TsType
+# from pyaerocom.variable_helpers import get_variable
```
Comment (thorbjoernl) [inline src/pyaro_readers/units_helpers.py]: Is this meant to be here?

**Review by thorbjoernl — CHANGES_REQUESTED** [review]

File: src/pyaro_readers/units_helpers.py (hunk 3 above)
Comment (jgriesfeller) [reply src/pyaro_readers/units_helpers.py]: removed

File: src/pyaro_readers/harpreader/harpreader.py (hunk 2 above)
Comment (jgriesfeller) [reply src/pyaro_readers/harpreader/harpreader.py]: changed to load_dataset()

**Pull request merged**